libblkid/samples/superblocks.c

```diff
@@ -44,7 +44,8 @@ int main(int argc, char *argv[])
 			BLKID_SUBLKS_UUID | BLKID_SUBLKS_UUIDRAW |
 			BLKID_SUBLKS_TYPE | BLKID_SUBLKS_SECTYPE |
 			BLKID_SUBLKS_USAGE | BLKID_SUBLKS_VERSION |
-			BLKID_SUBLKS_MAGIC | BLKID_SUBLKS_FSSIZE);
+			BLKID_SUBLKS_MAGIC | BLKID_SUBLKS_FSSIZE |
+			BLKID_SUBLKS_FSLASTBLOCK);
```

Comment:
This is probably good enough now for the patch, but maybe it would be better by a separate patch merge all the BLKID_SUBLKS_FSxxxx flags to the one BLKID_SUBLKS_FSINFO (or so) and also use it for BLOCK_SIZE.

Reply:
Sure, thanks, I will do that :)